app.rb

```diff
@@ -22,12 +22,16 @@ def initialize
     @compliance_data = ComplianceData.new
   end
 
+  cloudgov_path = File.join(root, 'node_modules', 'cloudgov-style')
   set :assets, Sprockets::Environment.new(root)
 
   configure do
     assets.append_path File.join(root, 'assets', 'stylesheets')
     assets.append_path File.join(root, 'assets', 'javascripts')
 
+    # cloudgov-style css and js. must be installed before running via `npm install`
```

Comment:
Where do you think would be best to add this check, in the app code itself? Or perhaps in `config.ru`?